### PR TITLE
fix type declarations

### PR DESCRIPTION
**Pull request by trusktr**
- [x] I've read the contributing guidelines

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -39,6 +39,8 @@ under the licensing terms detailed in LICENSE:
 * bnbarak <bn.barak@gmail.com>
 * Colin Eberhardt <colin.eberhardt@gmail.com>
 * Ryan Pivovar <ryanpivovar@gmail.com>
+* Joe Pea <trusktr@gmail.com>
+* Joe Pea <joe@trusktr.io>
```
Comment (MaxGraey) [inline NOTICE:43]: could you leave only one email? Which linked to Github profile only

Comment (MaxGraey) [reply NOTICE:43]:
```suggestion
* Joe Pea <trusktr@gmail.com>
```

Comment (MaxGraey) [reply NOTICE:43]:
```suggestion
* Joe Pea <trusktr@gmail.com>
```

Comment (MaxGraey) [reply NOTICE:43]: cc @trusktr

Comment (trusktr) [reply NOTICE:43]: I thought I'd put both in case I commit with one or the other.

Comment (trusktr) [reply NOTICE:43]: But I suppose I can try to fix my systems to have one email.

Comment (trusktr) [reply NOTICE]: Ok, I removed one.

Comment (MaxGraey) [reply NOTICE]: hmm, it seems wrong mail. Try `<joe@trusktr.io>`

Comment (trusktr) [reply NOTICE]: Huh, interesting. I verified I have both emails in my GitHub account. Updated to the other one.

**Comment by MaxGraey** [comment]
thanks!

**Pull request merged**